runtime/compiler/x/codegen/CallSnippet.cpp

```diff
@@ -480,7 +482,7 @@ TR_Debug::print(TR::FILE *pOutFile, TR::X86PicDataSnippet *snippet)
       printLabelInstruction(pOutFile, "jmp", snippet->getDoneLabel());
       bufferPos += 5;
 
-      if (methodSymRef->isUnresolved())
+      if (methodSymRef->isUnresolved() || fej9->forceUnresolvedDispatch())
```

Comment:
Actually in this case, the debug code would sometimes fail to print out the data snippet data under AOT (because it only checked the unresolved condition, and not the "is AOT" condition).